content/en/apps/concepts/interoperability.md

```diff
@@ -3,36 +3,55 @@ title: Interoperability
+
+# Loss to Follow Up Workflow
 
-The recommended approach is to use OpenHIM as the middleware component with [Mediators](http://openhim.org/docs/configuration/mediators/) to do the conversion. [Outbound Push]({{< ref "apps/reference/app-settings/outbound" >}}) should be configured to make a request to the middleware when relevant documents are created or modified in the CHT. A Mediator then calls [CHT APIs]({{< ref "apps/reference/api" >}}) to gather any additional data required to create a FHIR resource which is then routed to OpenHIM. OpenHIM will then route the resource to any other configured systems.
+This workflow describes a use case where a health facility/requesting system generates a list of patients who have missed appointments for follow-up through on the CHT. A Community Health Worker (CHW) physically visits the patients who missed the appointments and encourages them to visit the health facility. The follow-up process can also be conducted over phone or text messaging.
+In the context of the proof of concept, the intended users of the interoperable systems are Community Health Workers (CHWs) on the CHT side and Healthcare Givers on the requesting system side. System administrators can access the mediator on the administration console.
+
+The Interoperating Systems
+- [CHT](https://docs.communityhealthtoolkit.org/): Community Health Toolkit.
+- [OpenHIM](http://openhim.org/): Mediator (middleware).
+- Requesting System: [Postman](https://www.postman.com/) was used for testing purposes.
+
+OpenHIM was chosen as the main component of the interoperability layer and custom mediators were built to deal with the different workflows as it provides a central point of control for managing data exchange and security.
+
+{{< figure workflows="LTFU Sequence Diagram.png" link="LTFU Sequence Diagram.png" >}}
 
 Conversely to bring data in to the CHT, OpenHIM should be configured to route the updated resource to a Mediator, which then calls the relevant CHT APIs to update the document in the CHT database. This will then be replicated to users' devices as per usual.
 
-A reference application for this pattern is available in the [CHIS Interoperability repository](https://github.com/medic/chis-interoperability).
+# Important Links
+- A reference to an application repository can be accessed in the [CHT-interoperability repository](https://github.com/medic/cht-interoperability).
+- This [README](https://github.com/medic/cht-interoperability#readme) contains documentation on the CHT-interoperability project.
+- Test instances:
+- - [CHT](https://interop-cht-test.dev.medicmobile.org/medic/login?redirect=https%3A%2F%2Finterop-cht-test.dev.medicmobile.org%2F)
+- - [OpenHIM Admin Console](https://interoperability.dev.medicmobile.org/#!/login) 
```

Comment:
There's some repetition here, we can simplify (but admittedly add a bit of repetitiveness :shrug: )

```suggestion
-  [CHT-interoperability repository](https://github.com/medic/cht-interoperability): A reference application for the LTFU interoperability
- [CHT LTFU Test Instance](https://interop-cht-test.dev.medicmobile.org/medic/login?redirect=https%3A%2F%2Finterop-cht-test.dev.medicmobile.org%2F)
- [OpenHIM LTFU Test Instance Admin Login](https://interoperability.dev.medicmobile.org/#!/login) 
```